docs/sample.rst

```diff
@@ -35,38 +33,58 @@ optional arguments:
                         seconds (default: None)
   -n SHARE_NAME, --share-name SHARE_NAME
                         optionally, subscribe to a shared destination using
-                        SHARE_NAME as the share name.
+                        SHARE_NAMEas the share name.
   -f FILE, --file FILE  write the payload of the next message received to FILE
                         (overwriting previous file contents then end. (default
                         is to print messages to stdout)
   -d DELAY, --delay DELAY
-                        delay for DELAY seconds each time a message is
-                        received. (default: 0)
-  --verbose VERBOSE     print additional information about each message.
-                        (default: False)
+                        delays the confirmation for DELAY seconds each time a
+                        message is received. (default: 0)
+  --verbose             print additional information about each message.
+
+ssl arguments:
+  -c FILE, --trust-certificate FILE
+                        use the certificate contained in FILE (in PEM or DER
+                        format) to validate the identify of the server. The
+                        connection must be secured with SSL/TLS (e.g. the
+                        service URL must start with "amqps://")
+  --client-certificate  FILE
+                        use the certificate contained in FILE (in PEM format)
+                        to supply the identity of the client. The connection
+                        mustbe secured with SSL/TLS
+  --client-key FILE     use the private key contained in FILE (in PEM format)
+                        for encrypting the specified client certificate
+  --client-key-passphrase PASSPHRASE
+                        use PASSPHRASE to access the client private key
+  --no-verify-name SSL_VERIFY_NAME
+                        specify to not additionally check the server's common
+                        name in the specified trust certificate matches the
+                        actual server's DNS name
+
+
+.. index:: Sample;Sender
 
 Sender Sample:
-::
-usage: send.py [-h] [-s SERVICE] [-c TRUST_CERTIFICATE] [-t TOPIC]
-[-i CLIENT_ID] [--message-ttl MESSAGE_TTL] [-d DELAY] [-r REPEAT]
-[--sequence SEQUENCE] [-f FILE][MESSAGE [MESSAGE ...]]
 
-Send a message to an MQ Light server.
+usage: send.py [-h] [-s SERVICE] [-t TOPIC] [-i CLIENT_ID]
+               [--message-ttl MESSAGE_TTL] [-d DELAY] [-r REPEAT] [--sequence]
+               [-f FILE] [--verbose] [-c FILE] [--client-certificate  FILE]
+               [--client-key FILE] [--client-key-passphrase PASSPHRASE]
+               [--no-verify-name SSL_VERIFY_NAME]
+               [MESSAGE [MESSAGE ...]]
+
+
+Send a message to a MQ Light server.
 
 positional arguments:
-  MESSAGE               message to be sent (default: ['Hello world !'])
+  MESSAGE               message to be sent (default: None)
```

Comment:
As far as I can tell the send.py sample has not been changed and still sends "Hello world!" by default